packages/icons/src/img/sprite.svg

```diff
@@ -1983,5 +1983,10 @@
         clip-rule="evenodd"
       />
     </symbol>
+    <symbol viewBox="0 0 20 20" id="lp-icon-holdout">
```

Comment:
It would be awesome if you'd be willing to put this icon in alphabetical order like the rest of them! 🙏🏻

Reply:
Oh sorry about that, I will move into the correct order!